### PR TITLE
Creating separate file for docker and python log collection

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Create a dedicated documentation page for docker log collection
### Motivation

PM request
### Preview link
https://docs-staging.datadoghq.com/gus/log-docker/logs/docker/

### CODE REVIEW

**Comment by NBParis** [comment]
@l0k0ms Made a couple suggestions. I let you do the ultimate review

File: content/logs/docker.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  text: Learn more about parsing
+---
+
+<div class="alert alert-info">
+Datadog's Logs is currently available via public beta. You can apply for inclusion in the beta via <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+The Datadog Agent can collect logs from containers starting at the version 6. Two installations are possible:
```
Comment (jeremy-lq) [inline content/logs/docker.md]: Datadog Agent version 6 and greater can collect logs from containers.

File: content/logs/docker.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ 
+- on the host where the agent is external to the Docker environment
+- or by deploying its containerized version in the Docker environment
+
+## Setup
+### Host installation
+
+Install the [latest version of the agent 6](/logs/#getting-started-with-the-agent) on your host.
+
+The agent can both collect logs from [files on the host](/logs/#custom-log-collection) or from [container stdout/stderr](/logs/docker/#configuration-file-example). For this you need to update or create new .yaml configuration file in the agent’s `/conf.d` directory as explained in the provided links.
```
Comment (jeremy-lq) [inline content/logs/docker.md]: s/or create new/or create a new/

File: content/logs/docker.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Container installation
+
+As explained above, the agent also have a [containerized](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent) installation.
```
Comment (jeremy-lq) [inline content/logs/docker.md]: s/have/has/

File: content/logs/docker.md (hunk 1 above)
Comment (l0k0ms) [reply content/logs/docker.md]: 👍

File: content/logs/docker.md (hunk 2 above)
Comment (l0k0ms) [reply content/logs/docker.md]: 👍

File: content/logs/docker.md (hunk 3 above)
Comment (l0k0ms) [reply content/logs/docker.md]: 👍

**Pull request merged**